ts/services/ElementService.ts

```diff
@@ -179,6 +157,118 @@ class ElementService {
     return response;
   }
 
+  public async elementSearch(config, logger): Promise<void> {
+
+    // base query
+    const requestBody = this.buildBaseElementQuery(config);
+
+    // add name term
+    if (typeof config.name !== 'undefined') {
+      const field = 'elementNames';
```

Comment:
Why not directly reference the `elementNames` property in line 168 instead of making a temp variable? If you make the body object type `any` TypeScript won't complain.

Reply:
Not allowed by a lint rule: `[tslint] object access via string literals is disallowed (no-string-literal)`

Reply:
Yeah, I'm saying change it to `requestBody.body.elementNames` and change `const requestBody = this.buildBaseElementQuery(config);` to `const requestBody: any = this.buildBaseElementQuery(config);`